jinahub/indexers/compound/FaissPostgresIndexer/faisspsql.py

```diff
@@ -73,7 +73,12 @@ def __init__(
     def _init_executors(self, dump_path, kwargs, startup_sync_args):
         # float32 because that's what faiss expects
         kv_indexer = PostgreSQLStorage(dump_dtype=np.float32, **kwargs)
-        vec_indexer = FaissSearcher(dump_path=dump_path, prefetch_size=16, **kwargs)
+        vec_indexer = FaissSearcher(
+            dump_path=dump_path,
+            prefetch_size=16,
+            shard_id=self.runtime_args.pea_id,
```

Comment:
I'm confused about how this could happen. The **kwargs includes `runtime_args.pea_id`

Reply:
ah, I got a misunderstanding of `runtime_args`. This revision is not necessary.